x-pack/plugin/frozen-indices/src/internalClusterTest/java/org/elasticsearch/index/engine/frozen/FrozenIndexIT.java

```diff
@@ -329,9 +336,10 @@ public void testOpenPointInTimeWithNoIndexMatched() {
             );
             final String pitId = client().execute(OpenPointInTimeAction.INSTANCE, openPointInTimeRequest).actionGet().getPointInTimeId();
             try {
-                SearchResponse resp = prepareSearch().setPreference(null).setPointInTime(new PointInTimeBuilder(pitId)).get();
-                assertNoFailures(resp);
-                assertHitCount(resp, 0);
+                assertResponse(prepareSearch().setPreference(null).setPointInTime(new PointInTimeBuilder(pitId)), searchResponse -> {
```

Comment:
This case can use `assertNoFailuresAndResponse` to save us the `assertNoFailures(searchResponse);`, lets do that in all the spots we have here to keep the growing code size from this effort in check (at least a little)